data/BenceDv.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Bence Dvorszki",
+  "type": "personal",
+  "bio": "Avid **Florida** vibes enjoyer, working towards becoming a business man"
+  "avatar": "https://scontent-lcy1-1.cdninstagram.com/v/t51.2885-19/43107540_307581659844140_8731305201484431360_n.jpg?stp=dst-jpg_s320x320&_nc_ht=scontent-lcy1-1.cdninstagram.com&_nc_cat=108&_nc_ohc=lyK8vfCsV50AX8LGUNw&edm=AOQ1c0wBAAAA&ccb=7-5&oh=00_AfDZn7DnAXWZjUHk-8gdCVoH2s7ZhNR3BGQGHOzd-PKc_w&oe=6427CAC3&_nc_sid=8fd12b",
+  "links": [
+ {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/bencedv/",
+      "icon": "FaInstagram"
+    },
+{
+      "name": "Telegram",
+      "url": "https://t.me/bence2",
+      "icon": "FaPaperPlane"
+    },
+ {
+      "name": "Twitter",
+      "url": "https://twitter.com/BenceDvorszki",
+      "icon": "FaTwitter"
+    },
+{
+      "name": "Discord",
+      "url": "https://discord.gg/CX47T3p96X",
+      "icon": "FaDiscord"
+    },
```

Comment:
Please add a  square bracket &nbsp; **' ] '** &nbsp;&nbsp; between &nbsp; " } " &nbsp; and  &nbsp; " , "
```Suggestion
    }],
```